_src/app/_InProgressCacheMixin.js

```diff
@@ -113,7 +113,10 @@ define([
             // error: Error
             console.log('app/_InProgressCacheMixin:onError', arguments);
 
-            topic.publish(config.topics.toaster, 'Error ' + message + ' ' + error.message);
+            topic.publish(config.topics.toaster, {
+                type: 'danger',
+                message: 'Error ' + message + ' ' + error.message
```

Comment:
do you think it would be beneficial for bug reports to put more text here so we would know what topic was publishing this?